version.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
-  "version": "1.1",
+  "version": "2.2",
```

Comment:
Why bumping the version? Nothing was changed in the library code.

Reply:
I got confused with the fact that this "seems" to represent the schema vesion (rather than the nuget version). My bad... I will revert.